### DISCUSSION TITLE
[想法征集]如何对Apache中文网站进行改进（急需前端开发志愿者！）

### DISCUSSION THREAD

**Discussion by JesseZhou-1** [alc-beijing/SIG#6] [网站建设]
目前的[Apache中文网站](https://alc-beijing.github.io/alc-site/)还处于初级阶段，有很大改进空间。希望大家可以对于网站建设多提想法！
对网站已经前端感兴趣的小伙伴可以在此留言方便大家沟通交流。

**Comment by Ashira97** [comment]
前端开发志愿者举手报名~ 感觉文章的分类索引还有首页的指南应该要加一下~

**Reply by WillemJiang** [reply]
目前网站是使用[hugo](https://gohugo.io/)来生成的， 需要选择或者定制一个[主题](https://themes.gohugo.io/)。 欢迎对这块有想法的小伙伴加入讨论。

**Reply by Ashira97** [reply]
get~

**Comment by ruanwenjun** [comment]
Maybe we need to add a logo.

**Reply by WillemJiang** [reply]
[logo在此](https://cwiki.apache.org/confluence/display/COMDEV/ALC+Beijing)，只是感觉不太好放网站上。

**Comment by ousinka** [comment]
可以后台开发
